runtime/compiler/optimizer/JProfilingValue.cpp

```diff
@@ -454,168 +481,238 @@ TR_JProfilingValue::addProfilingTrees(
    if (roundedType == TR::Address)
       roundedType = systemType;
 
-   if (trace)
-      {
-      traceMsg(comp, "Inserting value profiling trees:\n  Value n%dn At n%dn\n  Table: %p\n",
-         value->getGlobalIndex(),
-         insertionPoint->getNode()->getGlobalIndex(),
-         table);
-      table->dumpInfo(comp->getOutFile());
-      }
-
    TR::Block *originalBlock = insertionPoint->getEnclosingBlock();
-   TR::Block *extendedBlock = originalBlock;
-   while (extendedBlock->isExtensionOfPreviousBlock())
-      extendedBlock = extendedBlock->getPrevBlock();
-
-   if (trace)
-      {
-      traceMsg(comp, "  Modifying block_%d", originalBlock->getNumber());
-      if (extendedBlock && extendedBlock != originalBlock)
-         traceMsg(comp, " extending block_%d", extendedBlock->getNumber());
-      traceMsg(comp, " with profiling\n");
-      }
-
-   // Cache the last treetop in the CFG
    TR::CFG *cfg = comp->getFlowGraph();
    cfg->setStructure(0);
-   TR::Block *cursor = originalBlock, *prev = originalBlock;
-   while (cursor && cursor != cfg->getEnd() && cursor != cfg->getStart() && cursor->getNumber())
+   /**
```

Comment:
I don't think this should be doxygen

Reply:
Fixed in https://github.com/eclipse/openj9/pull/6585/commits/4babc9149b99cc98d9703d5340c8c21f81c918da